display.css

```diff
@@ -0,0 +1,53 @@
+
+th{
```

Comment:
For all these styles, could you pls put spaces between the elements and the braces, for example:
```css
td {
 ...
}
```